src/components/Shared/Form/Form.js

```diff
@@ -2,28 +2,44 @@ import React, { useState, useEffect } from 'react';
 import { Link } from 'react-router-dom';
 import Input from '../Input/Input';
 import { useDispatch } from 'react-redux';
+import loginActions from '../../../store/login/action';
+import registerActions from '../../../store/register/action';
 
 function Form(props) {
-  const { title, inputArr, tertiaryButton, actions } = props;
+  const { title, inputArray, tertiaryButton } = props;
   const [isValid, setIsValid] = useState('');
   const [amountTrueArray, setAmountTrueArray] = useState([]);
 
   const dispatch = useDispatch();
 
   useEffect(() => {
-    dispatch(actions.logout());
+    if (title === 'Login') {
```

Comment:
Form should not really know about login. If you are splitting it out to be a reusable component it should not have logic that is specific to a feature.